terraform-docs/terraform-docs-module-examples.yml

```diff
@@ -0,0 +1,50 @@
+### To generate the output file to partially incorporate in the README.md,
+### Execute this command in the Terraform module's code folder:
+# terraform-docs -c .terraform-docs.yml .
+
+formatter: "markdown document" # this is required
+
+version: "~> 0.18"
+
+sections:
+  hide: []
+  show: []
+
+content: |-
+
+  {{ .Header }}
+
+  ```hcl
+  {{ include "main.tf" }}
+  ```
+  {{ .Outputs }}
+
+output:
+  file: README.md
+  mode: inject
```

Comment:
you do not need `inject` here, it's the default ;-)

ref. <https://terraform-docs.io/user-guide/configuration/output/>